Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[workspace]
```

Comment:
Do we have to litter the toplevel with this? I'd much rather have everything be contained under svix-cli.

Reply:
I wasn't able to figure out how to make it work without this. I'm sure it's possible, somehow.

Reply:
No worries then.

Reply:
I can ask around for clues (this wasn't a case described in the docs re: advanced layout/config) but we might end up being able to add bridge and so on into this workspace.

Could end up being an advantage to follow the happy path.

Reply:
yeah, let's not touch it. You're right.